templates/cli/lib/commands/push.js.twig

```diff
@@ -645,9 +681,100 @@ const createAttribute = async (databaseId, collectionId, attribute) => {
     }
 }
 
-const pushCollection = async ({ all, yes } = {}) => {
-    let response = {};
+const deleteAttribute = async (collection, attribute) => {
+    log(`Deleting attribute ${attribute.key} of ${collection.name} ( ${collection['$id']} )`);
 
+    await databasesDeleteAttribute({
+        databaseId: collection['databaseId'],
+        collectionId: collection['$id'],
+        key: attribute.key,
+        parseOutput: false
+    });
+}
+
+const deepSimilar = (remote, local, collection) => {
```

Comment:
Can we come up with a better name for this function ? It is not clear from its name what it does